### ISSUE TITLE
Feature request: additional documentation

### ISSUE THREAD

**Issue by KevinGrandon** [ImmoweltGroup/flow-mono-cli#87]
Tried to use this package, but setup documentation was scarce. Ideally looking for details on how to set this up with a new yarn workspace. Things that are currently not clear:

* Exactly which project to install this package into (root or sub-projects)
* How and when to execute this before running flow.

Ideally maybe there can be just a single command to run that replaces `yarn flow`?

**Comment by Inkdpixels** [comment]
Hey @KevinGrandon! Glad you found the time to try out this package! We are always happy to receive PRs of any sort, may it be a typo in the docs, improvements in 'em or even a new feature/bugfix, so if you are up for it, give it a shot! :)

> Exactly which project to install this package into (root or sub-projects)

Have your seen the [Mono-Repository Structure](https://immoweltgroup.gitbooks.io/flow-mono-cli/docs/introduction/MonoRepoStructure.html) section in the docs yet? In it you will find a recommended project structure as well as where the`flow-mono-cli` itself has to be installed. (The latter could **definitely** be emphasized a bit more since it is a bit hidden I guess 😄!)

**TL;DR** Only in the root of your mono-repo, sub-packages should not be aware that they are installed in a mono-repo since they (should) function as standalone packages.  :)

> How and when to execute this before running flow.

The `flow-mono-cli` is not intended to be run before `flow` or even as a replacement for the flow CLI itself.

See it more as a set of commands that help you bootstrap/setup your mono-repo to work with flow **if you face a specific problem**. I am not 100% sure if a "setup the project with all commands"-guide is the right approach since each command should only be used when facing a problem that the command itself tries to solve.

What do you think? What could we improve here? Again, we would love to see a PR as a foundation for discussions and on which we can build upon. :)

**Comment by KevinGrandon** [comment]
Thanks for the details, this will probably help folks. I think what might help more is example configs for common CI providers. E.g., show examples of this working on travis, circle, buildkite, etc, and I'm sure you will have many more customers.

**Comment by jeffersoneagley** [comment]
Some better installation instructions would be helpful. If you follow the current ones word-for-word you just end up with some stuff installed and no way to run it.
